lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -3,22 +3,17 @@
 
     define([
         'js/learner_dashboard/views/collection_list_view',
-        'js/learner_dashboard/views/sidebar_view',
+//        'js/learner_dashboard/views/sidebar_view',
         'js/learner_dashboard/views/program_card_view',
         'js/learner_dashboard/collections/program_collection'
     ],
-    function (CollectionListView, SidebarView, ProgramCardView, ProgramCollection) {
+    function (CollectionListView, ProgramCardView, ProgramCollection) {
         return function (options) {
             new CollectionListView({
                 el: '.program-cards-container',
                 childView: ProgramCardView,
                 collection: new ProgramCollection(options.programsData)
             }).render();
-
-            new SidebarView({
-                el: '.sidebar',
-                context: options
-            }).render();
```

Comment:
It looks like the approach you have taken is to move the sidebar into its own factory (so the page is loading 2 Backbone apps). Why was this decision made? I think the name of this file may be causing confusion.